ci: workflow to validate YAML schema of workflow files (#238)
asyncapi/.github: README.md

</h1>

The purpose of this repository is to keep all the community health files that we would normally have to duplicate in each repository. These are global community files that apply to repositories that do not have their own files.

Below is a list of workflows with their corresponding topics and descriptions:

> | Workflow Name | Topics Covered | Description |
> | --- | --- | --- |
> | get-workflows-validation | YAML schema validation | Validates the YAML schema of a workflow file |

Reviewer: ```suggestion
| Topic to include | Workflows you will get | Description |
| --- | --- | --- |
| `get-workflows-validation` | [validate-workflow-schema](.github/workflows/validate-workflow-schema.yml) | Validates the YAML schema of a workflow files in your repository |
```
What about this format? 🤔

Author: This looks cool with more clarity. I will update the rest of the documentation in a similar manner. Thank you, Khuda! :)
